test/intern/functional/Keyboard.js

```diff
@@ -1,14 +1,18 @@
 define([
-	'intern!tdd',
-	'intern/chai!assert',
-	'intern/dojo/node!leadfoot/helpers/pollUntil',
-	'intern/dojo/node!leadfoot/keys',
-	'require'
-], function (test, assert, pollUntil, keys, require) {
+	'dojo/node!@theintern/leadfoot/helpers/pollUntil',
+	'dojo/node!@theintern/leadfoot/keys'
+], function (pollUntilModule, keysModule) {
+	var test = intern.getPlugin('interface.tdd');
+	var assert = intern.getPlugin('chai').assert;
+	/* jshint -W024 */
+	var pollUntil = pollUntilModule.default;
```

Comment:
@jason0x43  is there a better way to handle getting the `default` export?

Reply:
Not that I'm aware of if you're loading tests with the dojo loader.

Reply:
Is there another loader that would be better to use? dgrid tests load AMD modules from dgrid itself, dojo, dstore, and dijit.

Reply:
You shouldn't need to use the dojo loader for your functional tests -- those only ever run in Node, and can just use the default node loader. If you wanted to use more modern import semantics in the unit tests, you could build the tests with webpack.